task_DEV_5/task_DEV_5/BubbleSorter.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.IO;
+using System.Text;
+
+namespace task_DEV_5
+{
+    class BubbleSorter
+    {
+        private string path;
+        private FileStream stream;
+
+        public BubbleSorter(string path)
+        {
+            this.path = path;
+        }
+
+        private string ReadString()
```

Comment:
Private methods should be one by one after public methods.

Reply:
Done.